CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 # PayPal iOS SDK Release Notes
 
 ## unreleased
+* Breaking Changes
+  * CardPayments
+    * Add `vault`property in `CardRequest`, optional `Vault` type that contains `customerID`, an optional String property  
```

Comment:
I had a question on this. It doesn't actually break anyone's implementation since it is an optional property set to nil by default in initializer.
Do I still put it as a breaking change?